### PR TITLE
Minor changes to fix identity page

### PR DESCRIPTION
**Pull request by lakshayman**

### CODE REVIEW

File: app/styles/identity.css
Comment (rohan09-raj) [inline app/styles/identity.css:1]: @lakshayman Can you please check that on hover, tooltip is breaking the changes
image

Comment (rohan09-raj) [reply app/styles/identity.css:1]: It's on `Save` and `Generate Chaincode` hover

File: app/templates/identity.hbs
Comment (rohan09-raj) [inline app/templates/identity.hbs:1]: Can we please put the fields in order of progress like first Generate Chaincode, then profile URL then checkbox

File: app/controllers/identity.js [hunk 1]
```diff
@@ -22,11 +22,34 @@ export default class IdentityController extends Controller {
...
+      (this.checkboxDisabled && type === 'CHECKBOX')
+    ) {
+      let error = 'Error: ';
+      if (type === 'CHECKBOX') {
+        if (!this.isChaincodeClicked) error += 'Generate Chaincode Not Clicked';
+        if (!this.isCopyClicked) error += ', Copy Button Not Clicked';
+      } else if (type === 'BUTTON') {
+        if (!this.isChaincodeClicked) error += 'Generate Chaincode Not Clicked';
+        if (!this.isCopyClicked) error += ', Copy Button Not Clicked';
+        if (!this.isChecked) error += ', Checkbox Not Clicked';
```
Comment (sumitd94) [inline app/controllers/identity.js:47]: Can we move these texts to constants and use it here?

**Review by shubhamsigdar1 — COMMENTED** [review]
Hello @lakshayman Can you please add one video in description
just show the UI working part.

**Review by shubhamsigdar1 — APPROVED** [review]

**Review by rohan09-raj — APPROVED** [review]

**Pull request merged**